src/vm-repair/setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "1.0.9"
+VERSION = "1.1.0"
 
```

Comment:
If the next version has breaking-change, version should be `2.0.0` here.

Reply:
Done.